File: app/src/main/java/com/plixapp/czirjak/plixapp/wilds/requests/WildResponse.java
package com.plixapp.czirjak.plixapp.wilds.requests;

import com.google.gson.annotations.SerializedName;

public class WildResponse {

    @SerializedName(value = "data")
    public WildsList wildsList;

    public WildResponse() {
    }
}

File: app/src/main/java/com/plixapp/czirjak/plixapp/wilds/requests/WildResponse.kt
package com.plixapp.czirjak.plixapp.wilds.requests

import com.google.gson.annotations.SerializedName
import com.plixapp.czirjak.plixapp.wilds.model.WildsList

class WildResponse {

    @SerializedName(value = "data")
    var wildsList: WildsList? = null
}
